### DISCUSSION TITLE
How to limit amount of things on current thread

### DISCUSSION THREAD

**Discussion by JSH32** [uNetworking/uWebSockets#1128] [Q&A]
I use highly thread based libraries that store thread_locals and other. How would I limit amount of things on one thread at one time for sockets

**Comment by ghost** [comment]
Generally speaking, threading is advanced use and then it is assumed you can manage things on your own.

You don't need threading, and if you opt to use it, then you are on your own (unless of course you can show bugs or lacking features).
